general/error.hpp

```diff
@@ -152,7 +152,11 @@ void mfem_warning(const char *msg = NULL);
       asm("trap;");            \
    }
 #else
-#define MFEM_ABORT_KERNEL(msg) MFEM_ABORT(msg)
+#define MFEM_ABORT_KERNEL(msg) \
+   {                           \
+      printf(msg);             \
+      abort();            \
+   }
```

Comment:
Are we sure we want that change?

Reply:
I ran into the same issue with ``MFEM_ABORT_KERNEL`` as @nbeams in PR https://github.com/mfem/mfem/pull/1714, but since she has a fix we should merge her PR before this one.